js/class.js

```diff
@@ -0,0 +1,78 @@
+const path = 'https://daniellamarr-fast-food-fast.herokuapp.com';
+
+class FastFood {
```

Comment:
#### Critical
Document your classes and methods using `jsdoc`
Check [here](http://usejsdoc.org/about-getting-started.html)

Reply:
Done